src/bytecode/generate.rs

```diff
@@ -783,12 +807,23 @@ impl BytecodeGenerator {
             .push(Bytecode::InvokeStaticPtr(dest, fid, start, num));
     }
 
+    fn generate_string_pool(&mut self) -> Vec<String> {
```

Comment:
I think this function should be defined as `fn generate_string_pool(map: HashMap<String, StrConstPoolIdx>) -> Vec<String>`